dotCMS/src/main/webapp/html/js/dotcms/dijit/form/FileAjaxUploader.jsp

```diff
@@ -1,29 +1,51 @@
 <%@ page import="com.liferay.portal.language.LanguageUtil" %>
 <div id="${id}" class="fileAjaxUploader">
+    <input name="${name}" id="${name}ValueField" dojoAttachPoint="fileNameField" type="hidden" value="${fileName}" />
+    <div dojoAttachPoint="fileUploadForm" class="fileAjaxUploaderForm">
+        <input
+            name="${name}FileUpload"
+            type="file"
+            accept="${accept}"
+            dojoAttachEvent="onchange:_doFileUpload"
+            dojoAttachPoint="fileInputField"
+            class="form-text"
+            size="30"
+        />
+    </div>
+    <div dojoAttachPoint="fileUploadStatus" class="fileAjaxUploaderStatus" style="display: none;">
+        <div dojoType="dijit.ProgressBar" dojoAttachPoint="progressBar"></div>
+        <div class="fileAjaxUploaderStatusMsg"><%= LanguageUtil.get(pageContext, "Uploading")%>...</div>
+    </div>
+    <div
+        name="${name}FileName"
+        dojoAttachPoint="fileNameDisplayField"
+        class="fileAjaxUploaderFileName"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Do not use an attribute called class.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5396940)